LICENSE

```diff
@@ -1,11 +1,27 @@
 MIT License
 
-Copyright (c) 2020 Crown Copyright, Government of Canada (Canadian Centre for Cyber Security / Communications Security Establishment)
+Copyright (c) 2025 Crown Copyright, Government of Canada
```

Comment:
```suggestion
Copyright (c) 2020 Crown Copyright, Government of Canada
```

Maintain the original copyright year?

Reply:
Since this project did not exist in 2020, and we are still updating it, I will keep it to 2025. I think it was a mistake in the first place. 😅